examples/CloudFunction/gsuite-exporter-cloudfunction/requirements.txt

```diff
@@ -0,0 +1,2 @@
+boto3
+gsuite_exporter==0.0.2
```

Comment:
Increment version to latest version: 0.0.3.
Ideally, make this file generated by Terraform and add a `gsuite_exporter_version` variable to the Terraform defaulting to the latest version.

Reply:
```suggestion
gsuite_exporter==0.0.3
```